setup.py

```diff
@@ -50,13 +57,14 @@ def run(self):
         Extension(
             name=f'natcap.invest.{package}.{module}',
             sources=[f'src/natcap/invest/{package}/{module}.pyx'],
-            include_dirs=[numpy.get_include()] + ['src/natcap/invest/managed_raster'],
-            extra_compile_args=compiler_args + compiler_and_linker_args,
+            include_dirs=include_dirs,
+            extra_compile_args=compiler_args + package_compiler_args + compiler_and_linker_args,
             extra_link_args=compiler_and_linker_args,
             language='c++',
+            libraries=['gdal'],
+            library_dirs=library_dirs,
```

Comment:
Oh neat! Thanks for pointing that out. I'm not sure what the solution would be for Windows, to avoid relying on conda - possibly we'd need to make the GDAL lib location configurable as an env variable